examples/onnxruntime/training/image-classification/run_image_classification.py

```diff
@@ -47,17 +49,16 @@
 from transformers.utils import check_min_version, send_example_telemetry
 from transformers.utils.versions import require_version
 
-from optimum.onnxruntime import ORTTrainer, ORTTrainingArguments
-
+from optimum import ORTTrainer, ORTTrainingArguments
 
 """ Fine-tuning a 🤗 Transformers model for image classification"""
 
 logger = logging.getLogger(__name__)
 
 # Will error if the minimal version of Transformers is not installed. Remove at your own risks.
-check_min_version("4.34.0")
+check_min_version("4.39.0.dev0")
```

Comment:
```suggestion
check_min_version("4.38.0")
```
Let's loosen a bit this constraint to the latest stable release of Transformers (released yesterday) otherwise this will force users to install Transformers from source.